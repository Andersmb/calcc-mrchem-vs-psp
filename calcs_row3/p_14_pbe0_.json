{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -3.6090379412692046
          ],
          "radius": 2.05
        },
        {
          "center": [
            0.0,
            0.0,
            0.8087634687307956
          ],
          "radius": 1.23
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "li",
          "xyz": [
            0.0,
            0.0,
            -3.6090379412692046
          ]
        },
        {
          "atom": "p",
          "xyz": [
            0.0,
            0.0,
            0.8087634687307956
          ]
        }
      ],
      "multiplicity": 3
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "p_14_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -7.706154786395868e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 2.689967197003839,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -1.3028826692453242e-06,
            -3.7673419256844783e-06,
            -2.689967197000885
          ],
          "vector_el": [
            -1.3029829585542758e-06,
            -3.767442283150028e-06,
            -3.9943054041465738
          ],
          "vector_nuc": [
            1.0028930895160545e-10,
            1.0035746554976596e-10,
            1.3043382071456886
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Li",
          "xyz": [
            0.0,
            0.0,
            -3.6090379412692046
          ]
        },
        {
          "symbol": "P",
          "xyz": [
            0.0,
            0.0,
            0.8087634687307956
          ]
        }
      ],
      "multiplicity": 3,
      "orbital_energies": {
        "energy": [
          -77.16316415812632,
          -6.559907306970183,
          -4.700398228427715,
          -4.700398222071693,
          -4.688275717972793,
          -2.041285067851667,
          -0.5343767526936882,
          -0.21229559141720908,
          -0.2122955891906201,
          -0.19919106919950505,
          -77.15531926674753,
          -6.551829026513698,
          -4.683812917528859,
          -4.683640974040216,
          -4.683640956992437,
          -2.0416301599190105,
          -0.4503325106110712,
          -0.17788244859980387
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -201.43967596487403
      },
      "scf_energy": {
        "E_ee": 169.02426260600927,
        "E_eext": 0.0,
        "E_el": -358.86544741311116,
        "E_en": -851.0827598302423,
        "E_kin": 348.1408845548289,
        "E_next": 0.0,
        "E_nn": 10.186062211429283,
        "E_nuc": 10.186062211429283,
        "E_tot": -348.6793852016819,
        "E_x": -6.110091484404499,
        "E_xc": -18.83774325930253,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 170.31514221267736,
        "E_eext": 0.0,
        "E_el": -357.01677192621605,
        "E_en": -848.3232365423055,
        "E_kin": 345.9788466068044,
        "E_next": 0.0,
        "E_nn": 10.186062211429283,
        "E_nuc": 10.186062211429283,
        "E_tot": -346.8307097147868,
        "E_x": -6.123907102343532,
        "E_xc": -18.863617101048778,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 168.34673510853787,
              "E_eext": 0.0,
              "E_el": -358.79989683570983,
              "E_en": -846.844436071884,
              "E_kin": 344.5501081568527,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.61383462428057,
              "E_x": -6.089861580355409,
              "E_xc": -18.762442448860916,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.61383462428057,
            "energy_update": 1.7831249094937789,
            "mo_residual": 0.28135829447068894,
            "wall_time": 223.459570009
          },
          {
            "energy_terms": {
              "E_ee": 169.00266421858404,
              "E_eext": 0.0,
              "E_el": -358.8550803076578,
              "E_en": -849.5478109901329,
              "E_kin": 346.6123463211045,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6690180962285,
              "E_x": -6.105212693409111,
              "E_xc": -18.8170671638043,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6690180962285,
            "energy_update": 0.0551834719479416,
            "mo_residual": 0.09945007024464893,
            "wall_time": 163.483730861
          },
          {
            "energy_terms": {
              "E_ee": 168.92483056361556,
              "E_eext": 0.0,
              "E_el": -358.8627753097937,
              "E_en": -850.2699349513096,
              "E_kin": 347.41177634654923,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6767130983644,
              "E_x": -6.106305659457599,
              "E_xc": -18.823141609191346,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6767130983644,
            "energy_update": 0.007695002135903906,
            "mo_residual": 0.04708103843320011,
            "wall_time": 125.293124222
          },
          {
            "energy_terms": {
              "E_ee": 169.01555204312163,
              "E_eext": 0.0,
              "E_el": -358.86449396825736,
              "E_en": -850.74178136185,
              "E_kin": 347.8030373089749,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6784317568281,
              "E_x": -6.108836173202142,
              "E_xc": -18.83246578530173,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6784317568281,
            "energy_update": 0.0017186584636874613,
            "mo_residual": 0.024778341748876394,
            "wall_time": 58.106945829
          },
          {
            "energy_terms": {
              "E_ee": 169.05771852453614,
              "E_eext": 0.0,
              "E_el": -358.8652342409557,
              "E_en": -851.2976025428135,
              "E_kin": 348.32708115328114,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.67917202952646,
              "E_x": -6.1110920938449516,
              "E_xc": -18.841339282114582,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.67917202952646,
            "energy_update": 0.0007402726983514185,
            "mo_residual": 0.03241201086267265,
            "wall_time": 55.977597737
          },
          {
            "energy_terms": {
              "E_ee": 169.04675220948528,
              "E_eext": 0.0,
              "E_el": -358.865324560413,
              "E_en": -851.2622297882485,
              "E_kin": 348.3016162283595,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.67926234898374,
              "E_x": -6.110829779141158,
              "E_xc": -18.840633430868078,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.67926234898374,
            "energy_update": 9.031945728565915e-05,
            "mo_residual": 0.012178876656773857,
            "wall_time": 55.858691982
          },
          {
            "energy_terms": {
              "E_ee": 169.03177542310746,
              "E_eext": 0.0,
              "E_el": -358.8654219852853,
              "E_en": -851.0704103323936,
              "E_kin": 348.12125893994147,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.67935977385605,
              "E_x": -6.110139441897482,
              "E_xc": -18.83790657404314,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.67935977385605,
            "energy_update": 9.742487230823826e-05,
            "mo_residual": 0.010615795280637351,
            "wall_time": 56.488582876
          },
          {
            "energy_terms": {
              "E_ee": 169.0269763457112,
              "E_eext": 0.0,
              "E_el": -358.86543605515396,
              "E_en": -851.053184785533,
              "E_kin": 348.1083413061814,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6793738437247,
              "E_x": -6.110045673613227,
              "E_xc": -18.83752324790037,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6793738437247,
            "energy_update": 1.4069868655042228e-05,
            "mo_residual": 0.0034763599066476103,
            "wall_time": 58.765733991
          },
          {
            "energy_terms": {
              "E_ee": 169.0219070555589,
              "E_eext": 0.0,
              "E_el": -358.86544234652814,
              "E_en": -851.0719407004187,
              "E_kin": 348.13212546564574,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6793801350989,
              "E_x": -6.11002507473488,
              "E_xc": -18.837509092579126,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6793801350989,
            "energy_update": 6.291374177180842e-06,
            "mo_residual": 0.003875762768603015,
            "wall_time": 57.368958756
          },
          {
            "energy_terms": {
              "E_ee": 169.02188152798573,
              "E_eext": 0.0,
              "E_el": -358.8654451352368,
              "E_en": -851.078414798327,
              "E_kin": 348.13875482489794,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.67938292380757,
              "E_x": -6.110063537972308,
              "E_xc": -18.837603151821234,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.67938292380757,
            "energy_update": 2.7887086844202713e-06,
            "mo_residual": 0.0016120056581779491,
            "wall_time": 58.676012743
          },
          {
            "energy_terms": {
              "E_ee": 169.0242901970886,
              "E_eext": 0.0,
              "E_el": -358.8654467960943,
              "E_en": -851.0883226018593,
              "E_kin": 348.1465224045694,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.67938458466506,
              "E_x": -6.110119478880643,
              "E_xc": -18.837817317012387,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.67938458466506,
            "energy_update": 1.6608574924248387e-06,
            "mo_residual": 0.0010663189701351254,
            "wall_time": 59.990040075
          },
          {
            "energy_terms": {
              "E_ee": 169.025143106545,
              "E_eext": 0.0,
              "E_el": -358.865447110409,
              "E_en": -851.0875879265361,
              "E_kin": 348.14495415510964,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6793848989797,
              "E_x": -6.1101195338414,
              "E_xc": -18.837836911686086,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6793848989797,
            "energy_update": 3.143146614092984e-07,
            "mo_residual": 0.0007762159076846265,
            "wall_time": 59.830032236
          },
          {
            "energy_terms": {
              "E_ee": 169.0250172445187,
              "E_eext": 0.0,
              "E_el": -358.8654472376139,
              "E_en": -851.0845142754868,
              "E_kin": 348.1419532762957,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.67938502618466,
              "E_x": -6.110106085945155,
              "E_xc": -18.83779739699638,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.67938502618466,
            "energy_update": 1.2720494169116137e-07,
            "mo_residual": 0.000546492951243775,
            "wall_time": 59.208214047
          },
          {
            "energy_terms": {
              "E_ee": 169.02438788247608,
              "E_eext": 0.0,
              "E_el": -358.86544738202747,
              "E_en": -851.0816388020855,
              "E_kin": 348.13962645877234,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6793851705982,
              "E_x": -6.11008692800619,
              "E_xc": -18.837735993184133,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6793851705982,
            "energy_update": 1.444135477868258e-07,
            "mo_residual": 0.0004089861223362039,
            "wall_time": 61.13291383
          },
          {
            "energy_terms": {
              "E_ee": 169.0241784073027,
              "E_eext": 0.0,
              "E_el": -358.8654474042378,
              "E_en": -851.0816832181479,
              "E_kin": 348.1398695418955,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6793851928085,
              "E_x": -6.11008529436863,
              "E_xc": -18.837726840919498,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6793851928085,
            "energy_update": 2.2210315364645794e-08,
            "mo_residual": 0.00014507590542528056,
            "wall_time": 60.517831356
          },
          {
            "energy_terms": {
              "E_ee": 169.02426260600927,
              "E_eext": 0.0,
              "E_el": -358.86544741311116,
              "E_en": -851.0827598302423,
              "E_kin": 348.1408845548289,
              "E_next": 0.0,
              "E_nn": 10.186062211429283,
              "E_nuc": 10.186062211429283,
              "E_tot": -348.6793852016819,
              "E_x": -6.110091484404499,
              "E_xc": -18.83774325930253,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -348.6793852016819,
            "energy_update": 8.873371371009853e-09,
            "mo_residual": 6.563935075469905e-05,
            "wall_time": 68.71433936
          }
        ],
        "wall_time": 1282.937255237
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
