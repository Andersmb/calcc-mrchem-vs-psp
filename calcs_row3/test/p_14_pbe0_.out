                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000       -3.609038
    1       P            :        0.000000        0.000000        0.808763
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              10
 Beta electrons          :                                               8
 Total electrons         :                                              18
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999999692e-01
    1    1    a          :                              9.999999999683e-01
    2    1    a          :                              1.000000001622e+00
    3    1    a          :                              1.000000001038e+00
    4    1    a          :                              1.000000001038e+00
    5    1    a          :                              9.999994135970e-01
    6    1    a          :                              9.999999999576e-01
    7    1    a          :                              1.000000002126e+00
    8    1    a          :                              9.999999999658e-01
    9    1    a          :                              9.999999999658e-01
   10    1    b          :                              9.999999999080e-01
   11    1    b          :                              1.000000000157e+00
   12    1    b          :                              9.999999999638e-01
   13    1    b          :                              1.000000000006e+00
   14    1    b          :                              1.000000000006e+00
   15    1    b          :                              9.999999548005e-01
   16    1    b          :                              9.999999999425e-01
   17    1    b          :                              9.999999948557e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                345.978846606804
 E-N energy              :            (au)               -848.323236542305
 Coulomb energy          :            (au)                170.315142212677
 Exchange energy         :            (au)                 -6.123907102344
 X-C energy              :            (au)                -18.863617101049
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 10.186062211429
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -357.016771926216
 Nuclear energy          :            (au)                 10.186062211429
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.468307097148e+02
                         :      (kcal/mol)             -2.176395544860e+05
                         :        (kJ/mol)             -9.106038965605e+05
                         :            (eV)             -9.437743989214e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.485746296354
    1    1    a          :            (au)                 -6.380142332257
    2    1    a          :            (au)                 -4.528007269864
    3    1    a          :            (au)                 -4.527994587255
    4    1    a          :            (au)                 -4.511407659255
    5    1    a          :            (au)                 -2.062294453150
    6    1    a          :            (au)                 -0.461942175942
    7    1    a          :            (au)                 -0.153894178981
    8    1    a          :            (au)                 -0.153890916039
    9    1    a          :            (au)                 -0.135570176836
   10    1    b          :            (au)                -76.477653208859
   11    1    b          :            (au)                 -6.371006648358
   12    1    b          :            (au)                 -4.510087888345
   13    1    b          :            (au)                 -4.510082155922
   14    1    b          :            (au)                 -4.506842501057
   15    1    b          :            (au)                 -2.061881641230
   16    1    b          :            (au)                 -0.380029501233
   17    1    b          :            (au)                 -0.106108688544
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -198.324582279479
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.242641e+00        -346.830709714787         -3.468307e+02
    1          2.813583e-01        -348.613834624281         -1.783125e+00
    2          9.945007e-02        -348.669018096229         -5.518347e-02
    3          4.708104e-02        -348.676713098364         -7.695002e-03
    4          2.477834e-02        -348.678431756828         -1.718658e-03
    5          3.241201e-02        -348.679172029526         -7.402727e-04
    6          1.217888e-02        -348.679262348984         -9.031946e-05
    7          1.061580e-02        -348.679359773856         -9.742487e-05
    8          3.476360e-03        -348.679373843725         -1.406987e-05
    9          3.875763e-03        -348.679380135099         -6.291374e-06
   10          1.612006e-03        -348.679382923808         -2.788709e-06
   11          1.066319e-03        -348.679384584665         -1.660857e-06
   12          7.762159e-04        -348.679384898980         -3.143147e-07
   13          5.464930e-04        -348.679385026185         -1.272049e-07
   14          4.089861e-04        -348.679385170598         -1.444135e-07
   15          1.450759e-04        -348.679385192809         -2.221032e-08
   16          6.563935e-05        -348.679385201682         -8.873371e-09
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000       -3.609038
    1       P            :        0.000000        0.000000        0.808763
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                348.140884554829
 E-N energy              :            (au)               -851.082759830242
 Coulomb energy          :            (au)                169.024262606009
 Exchange energy         :            (au)                 -6.110091484404
 X-C energy              :            (au)                -18.837743259303
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 10.186062211429
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -358.865447413111
 Nuclear energy          :            (au)                 10.186062211429
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.486793852017e+02
                         :      (kcal/mol)             -2.187996158592e+05
                         :        (kJ/mol)             -9.154575933488e+05
                         :            (eV)             -9.488049009720e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.163164158126
    1    1    a          :            (au)                 -6.559907306970
    2    1    a          :            (au)                 -4.700398228428
    3    1    a          :            (au)                 -4.700398222072
    4    1    a          :            (au)                 -4.688275717973
    5    1    a          :            (au)                 -2.041285067852
    6    1    a          :            (au)                 -0.534376752694
    7    1    a          :            (au)                 -0.212295591417
    8    1    a          :            (au)                 -0.212295589191
    9    1    a          :            (au)                 -0.199191069200
   10    1    b          :            (au)                -77.155319266748
   11    1    b          :            (au)                 -6.551829026514
   12    1    b          :            (au)                 -4.683812917529
   13    1    b          :            (au)                 -4.683640974040
   14    1    b          :            (au)                 -4.683640956992
   15    1    b          :            (au)                 -2.041630159919
   16    1    b          :            (au)                 -0.450332510611
   17    1    b          :            (au)                 -0.177882448600
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -201.439675964874
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000001       -0.000004       -3.994305
 Magnitude               :            (au)                        3.994305
                         :         (Debye)                       10.152511
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.304338
 Magnitude               :            (au)                        1.304338
                         :         (Debye)                        3.315297
---------------------------------------------------------------------------
 Total vector            :       -0.000001       -0.000004       -2.689967
 Magnitude               :            (au)                        2.689967
                         :         (Debye)                        6.837214
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 25m  9s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
